meson.build

```diff
@@ -427,6 +427,15 @@ You can disable the Wayland demo programs with -Denable-wayland=false.''')
     executable('interactive-wayland', 'test/interactive-wayland.c', xdg_shell_sources, dependencies: [test_dep, wayland_client_dep])
 endif
 
+# xkeyboard-config "verifier"
+xkct_config = configuration_data()
+xkct_config.set('MESON_BUILD_ROOT', meson.build_root())
+xkct_config.set('XKB_CONFIG_ROOT', XKBCONFIGROOT)
+configure_file(input: 'test/xkeyboard-config-test.py',
+               output: 'xkeyboard-config-test.py',
```

Comment:
The fact that it's a python script is not particularly interesting to the user, so I'd remove the `.py` from the output file.